src/pks/surface_balance/constitutive_relations/land_cover/LandCover.cc

```diff
@@ -133,10 +133,14 @@ checkValid(const std::string& region, const LandCover& lc, const std::string& pa
     throwInvalid(region, "snow transition depth [m]");
   if (parname == "water_transition_depth" && std::isnan(lc.water_transition_depth))
     throwInvalid(region, "water transition depth [m]");
-  if (parname == "dessicated_zone_thickness" && std::isnan(lc.dessicated_zone_thickness))
-    throwInvalid(region, "dessicated zone thickness [m]");
-  if (parname == "clapp_horn_b" && std::isnan(lc.clapp_horn_b))
-    throwInvalid(region, "Clapp and Hornberger b [-]");
+
+  if (parname == "rs_method" && lc.rs_method == std::string("sakagucki_zeng")) {
```

Comment:
I don't think this can ever get through to throwInvalid?  How can parname be both == "rs_method" and == "dessicated_zone_thickness"...

I think this should just be:

    if (parname == "dessicated_zone_thickness" && 
        std::isnan(lc.dessicated_zone_thickness) && 
        lc.rs_method == std::string("sakagucki_zeng")) {
            thowInvalid(region, "dessicated_zone_thickness");
    ...